client/components/Profile/Password.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { ReactElement, useState } from "react";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import { useIntl } from "react-intl";
+import API from "../../util/api";
+import useProfileStyles from "./Profile.styles";
+import useStyles from "./Password.styles";
+import { validatePassword } from "../Authentication/SignUp.service";
+
+const Password = (): ReactElement => {
+  const { formatMessage: _t } = useIntl();
+  const profileClasses = useProfileStyles({});
+  const classes = useStyles({});
+  const [oldPassword, setOldPassword] = useState("");
+  const [newPassword, setNewPassword] = useState("");
+  const [confirmedPassword, setConfirmedPassword] = useState("");
+  const [oldPasswordError, setOldPasswordError] = useState("");
+  const [newPasswordError, setNewPasswordError] = useState("");
+  const [confirmedPasswordError, setConfirmedPasswordError] = useState("");
+  const [status, setStatus] = useState();
```

Comment:
type status as a number
```suggestion
  const [status, setStatus] = useState<number>();
```